ToggleLockBranch/ReadMe.md

```diff
@@ -0,0 +1,55 @@
+# ToggleLockBranch
+
+Modify branch protection rules to Lock or Unlock configured branch.
+
+Locked branches cannot be modified, preventing pull request from being merged.
+
+Intended usage is to prevent any changes on a repository that is about to be released. And development team needs to lock it temporarily.
```

Comment:
```suggestion
The intended usage is to prevent any changes on a repository that is about to be released when the development team needs to lock it temporarily.
```